fv/workload/workload.go

```diff
@@ -85,8 +110,38 @@ func (w *Workload) Configure(client *client.Client) {
 	wep.Metadata.Workload = w.Name
 	wep.Metadata.Orchestrator = "felixfv"
 	wep.Metadata.Node = w.C.Hostname
+	wep.Metadata.Labels = map[string]string{"name": w.Name}
 	wep.Spec.IPNetworks = []net.IPNet{net.MustParseNetwork(w.IP + "/32")}
 	wep.Spec.InterfaceName = w.InterfaceName
+	wep.Spec.Profiles = []string{"default"}
 	_, err := client.WorkloadEndpoints().Create(wep)
 	Expect(err).NotTo(HaveOccurred())
 }
+
+func (w *Workload) CanConnectTo(ip, port string) bool {
+
+	// Ensure that the host has the 'test-connection' binary.
+	w.C.EnsureBinary("test-connection")
+
+	// Run 'test-connection' to the target.
+	connectionCmd := exec.Command("docker", "exec", w.C.Name,
+		"/test-connection", w.namespacePath, ip, port)
+	outPipe, err := connectionCmd.StdoutPipe()
+	Expect(err).NotTo(HaveOccurred())
+	errPipe, err := connectionCmd.StderrPipe()
+	Expect(err).NotTo(HaveOccurred())
+	err = connectionCmd.Start()
+	Expect(err).NotTo(HaveOccurred())
+
+	wOut, err := ioutil.ReadAll(outPipe)
+	Expect(err).NotTo(HaveOccurred())
+	wErr, err := ioutil.ReadAll(errPipe)
+	Expect(err).NotTo(HaveOccurred())
+	err = connectionCmd.Wait()
```

Comment:
Would it be worth using a specific return code for "connection failed" and checking for it here?  I'm thinking that would make it easier to spot if the test binary was failing for an unexpected reason (such as passing it bad arguments).

Reply:
I think I'd rather leave this one for future refinement, given current task pressure.

Reply:
Sure, no problem.  All of these were enhancement ideas rather than must haves